README.md

```diff
@@ -122,12 +122,48 @@ npm run dev
 ```
 
 **2. Disable public channels. Default: false**
-- Update `ABLY_DISABLE_PUBLIC_CHANNELS`, set as **true** in `.env` file. 
-- Update `ably` section under `config/broadcasting.php` with `'disable_public_channels' => env('ABLY_DISABLE_PUBLIC_CHANNELS', false)`
+- Set `ABLY_DISABLE_PUBLIC_CHANNELS` as **true** in **.env** file.
+```dotenv
+    ABLY_DISABLE_PUBLIC_CHANNELS=true
+```
+- Update ably section under `config/broadcasting.php` with
+```php
+        'ably' => [
+            'driver' => 'ably',
+            'key' => env('ABLY_KEY'),
+            'disable_public_channels' => env('ABLY_DISABLE_PUBLIC_CHANNELS', false)
+        ],
+```
 
 **3. Update token expiry. Default: 3600 seconds (1 hr)**
-- Update `ABLY_TOKEN_EXPIRY` in `.env` file. 
-- Update `ably` section under `config/broadcasting.php` with `'token_expiry' => env('ABLY_TOKEN_EXPIRY', 3600)`
+- Set `ABLY_TOKEN_EXPIRY` in **.env** file.
+```dotenv
+    ABLY_TOKEN_EXPIRY=21600
+```
+- Update ably section under `config/broadcasting.php` with
+```php
+        'ably' => [
+            'driver' => 'ably',
+            'key' => env('ABLY_KEY'),
+            'token_expiry' => env('ABLY_TOKEN_EXPIRY', 3600)
+        ],
+```
+
+**4. Use internet time for issued token expiry. Default: false**
+- Internet time in UTC format is fetched from `ably server` and cached every 6 hrs.
+- Set `ABLY_USE_INTERNET_TIME` as **true** in **.env** file.
+```dotenv
+    ABLY_USE_INTERNET_TIME=true
+```
+- Update ably section under `config/broadcasting.php` with
+```php
+        'ably' => [
+            'driver' => 'ably',
+            'key' => env('ABLY_KEY'),
+            'use_internet_time' => env('ABLY_USE_INTERNET_TIME', false)
```

Comment:
I think `query_time` is a better name for this, 'internet time' just sounds confusing in my opinion

Reply:
`QUERY_TIME` feels like a technical term. It's hard to understand the intent. 
I think `ABLY_USE_INTERNET_TIME` makes sense while reading. 

Maybe it would make sense to use `ABLY_SYNC_SERVER_TIME` instead, it's self-descriptive. 

We will have to take a look at the doc for `QUERY_TIME` to truly understand it.

Reply:
I think `ABLY_SYNC_SERVER_TIME` sounds better than `ABLY_USE_INTERNET_TIME`. I think `QUERY_TIME` is good in that it is aligned with what we call it everywhere else, but I do take your point about it being a bit too technical for laravel config.

Reply:
Updated, thanks 👍